core/modules/url_filter.cc

```diff
@@ -135,7 +135,9 @@ inline static bess::Packet *GenerateResetPacket(
 pb_error_t UrlFilter::Init(const bess::pb::UrlFilterArg &arg) {
   for (const auto &url : arg.blacklist()) {
     if (blacklist_.find(url.host()) == blacklist_.end()) {
-      blacklist_.emplace(url.host(), Trie());
+      blacklist_.emplace(std::piecewise_construct,
```

Comment:
Oh I just realized that it would be non-trivial, given that `Node` itself is not safe for copy at the moment. To make `Trie` RAII we will need recursive copy. I am fine with making it non-copyable if there is no urgent demand for copy constructors.